GitDiffMargin/Core/MarginCore.cs

```diff
@@ -13,36 +13,37 @@ namespace GitDiffMargin.Core
 {
     internal sealed class MarginCore : IMarginCore, IDisposable
     {
-        private readonly IWpfTextView _textView;
-
         private readonly IClassificationFormatMap _classificationFormatMap;
         private readonly IEditorFormatMap _editorFormatMap;
-        private readonly IGitCommands _gitCommands;
 
         private readonly DiffUpdateBackgroundParser _parser;
 
         private Brush _additionBrush;
-        private Brush _modificationBrush;
-        private Brush _removedBrush;
 
         private bool _isDisposed;
+        private Brush _modificationBrush;
```

Comment:
📝 Situations like this are the reason I have the following reaction when people suggest creating rules to alphabetize things:

# 😑